specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +789,115 @@
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Private dns zone group resource."
+    },
+    "PrivateDNSZoneGroupProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the private dns zone group resource."
+        },
+        "privateDnsZoneConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateDnsZoneConfig"
+          },
+          "description": "A collection of private dns zone configurations of the private dns zone group."
+        }
+      },
+      "description": "Properties of the private dns zone group."
+    },
+    "PrivateDnsZoneConfig": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZoneConfigProperties",
+          "description": "Properties of the private dns zone configuration."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "PrivateDnsZoneConfig resource."
+    },
+    "PrivateDnsZoneConfigProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the private dns zone group resource."
+        },
+        "privateDnsZoneId": {
+          "type": "string",
+          "description": "The resource id of the private dns zone."
+        }, 
+        "recordSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/recordSet"
+          },
+          "description": "A collection of information regarding a recordSet, holding information to identify private resources"
+        }
+      }
+    },
+    "recordSet": {
+      "properties": {
+        "recordType": {
+          "type": "string",
+          "description": "resource record type"
+        },
+        "recordSetName": { 
+          "type": "string",
+          "description": "recordset name"
+        },
+        "fqdn": { 
+          "type": "string",
+          "description": "fqdn of the recordset"
+        },
+        "ipAddresses": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The ip addresses this record set contains"
```

Comment:
should it be the private ip of private endpoint?

Reply:
Following up with manager, after review of the doc you sent, I believe so since this is how you'll connect to your DNS resource